tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -238,6 +238,12 @@ bgp/test_bgp_bbr_default_state.py::test_bbr_disabled_constants_yml_default:
     conditions:
       - "'-v6-' in topo_name"
 
+bgp/test_bgp_bounce.py:
+  xfail:
+    reason: "xfail for IPv6-only topologies, issue https://github.com/sonic-net/sonic-mgmt/issues/20753"
+    conditions:
+      - "https://github.com/sonic-net/sonic-mgmt/issues/20753 and '-v6-' in topo_name"
```

Comment:
Using `'-v6-' in topo_name` is not really the right way to tell if the DUT is running in IPv6 only mode.
Need to enhance the conditional mark to support this scenario: https://github.com/sonic-net/sonic-mgmt/issues/21125